modules/configuring-the-codeflare-operator.adoc

```diff
@@ -0,0 +1,102 @@
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+
+.Procedure
+ifdef::upstream,self-managed[]
+. In the {openshift-platform} console, click *Workloads* -> *ConfigMaps*.
+endif::[]
+ifdef::cloud-service[]
+. In the OpenShift console, click *Workloads* -> *ConfigMaps*.
+endif::[]
+
+ifdef::self-managed,cloud-service[]
+. From the *Project* list, select *redhat-ods-applications*.
+endif::[]
+ifdef::upstream[]
+. From the *Project* list, select *odh*.
+endif::[]
+
+. Search for the *codeflare-operator-config* config map, and click the config map name to open the *ConfigMap details* page.
+
+. Click the *YAML* tab to show the config map specifications.
+. In the `*data*` > `*config.yaml*` > `*kuberay*` section, you can edit the following entries:
++
+`*ingressDomain*`::
+This configuration option is null (`ingressDomain: ""`) by default.
+This option should remain null unless the Ingress Controller is not running on OpenShift.
+{productname-short} uses this value to generate the dashboard and client routes for every Ray Cluster, as shown in the following examples:
++
+.Example dashboard and client routes
+[source,bash,subs="+quotes"]
+----
+ray-dashboard-_<clustername>_-_<namespace>_._<your.ingress.domain>_
+ray-client-_<clustername>_-_<namespace>_._<your.ingress.domain>_
+----
++
+`*mTLSEnabled*`::
+This configuration option is enabled (`mTLSEnabled: true`) by default.
+When this option is enabled, the Ray Cluster pods create certificates that are used for mutual Transport Layer Security (mTLS), a form of mutual authentication, between Ray Cluster nodes.
+When this option is enabled, Ray clients cannot connect to the Ray head node unless they download the generated certificates from the `ca-secret-_<cluster_name>_` secret, generate the necessary certificates for mTLS communication, and then set the required Ray environment variables.
+The Ray clients must then be re-initialized to apply the changes.
+The CodeFlare SDK provides the following functions to simplify the authentication process for Ray clients:
++
+.Example Ray client authentication code
+[source,bash,subs="+quotes"]
+----
+from codeflare_sdk import generate_cert
+
+generate_cert.generate_tls_cert(cluster.config.name, cluster.config.namespace)
+generate_cert.export_env(cluster.config.name, cluster.config.namespace)
+
+ray.init(cluster.cluster_uri())
+----
+
++
+`*rayDashboardOauthEnabled*`::
+This configuration option is enabled (`rayDashboardOAuthEnabled: true`) by default.
+When this option is enabled, {productname-short} places an OpenShift OAuth proxy in front of the Ray Cluster head node.
+Users must then authenticate via their OpenShift cluster login credentials when accessing the Ray Dashboard through the browser.
```

Comment:
change **via** to **by using** as per IBM style guide: 
"Use this term, which means by way of, to refer to routing data, such as in “Messages are routed via SMTP”. In other situations, try to replace via with one of the following synonyms: across, along, by, from, on, or through."